src/actions/common.js

```diff
@@ -63,6 +65,33 @@ export const addUser = user => (dispatch) => {
   })
 }
 
+export const addNewUser = userId => (dispatch, getState) => {
+  const org = orgSelector(getState())
+
+  return api
+    .openPm(org.id, userId)
+    .then(({id, users}) =>
+      Promise.all([users, api.getChannel(id)])
```

Comment:
I would either put it onto same line or add braces ... need to start using prettier and verify this with eslint evtl